jekyll/_cci2/deploy-to-artifactory.md

```diff
@@ -39,7 +39,7 @@ Add the following to your `.circleci/config.yml`:
 ```yml
 - run:
     name: Install jFrog CLI
-    command: curl -fL https://getcli.jfrog.io | sh
+    command: curl -fL https://getcli.jfrog.io/v2-jf | sh
```

Comment:
Looks like this should be `curl -fL https://install-cli.jfrog.io | sh`? https://docs.jfrog-applications.jfrog.io/jfrog-applications/jfrog-cli/install#install-with-curl

Reply:
I believe `getcli` is the command we want in this case.  It mimics what was originally in the docs, and seems to be the approach prescribed in this blog:
https://github.com/jfrog/jfrog-cli/blob/v2/guides/getting-started-with-jfrog-using-the-cli.md

I can also share CircleCI config I successfully used this command in if that helps(private repo).

Reply:
@rosieyohannan I know with the holiday things might have dragged a bit, wondering if you've had a chance to review my last comment?

Reply:
@rosieyohannan Any update?

Reply:
@rosieyohannan Can you take a minute to re-review when you have a chance?

Reply:
Apologies, I missed hitting the re-review request button.  Something we started enforcing in our own team recently because otherwise it's just too difficult to sort through the open PR's.

Reply:
@rosieyohannan ^^.

Reply:
Should we instead link directly to their install docs? https://docs.jfrog-applications.jfrog.io/jfrog-applications/jfrog-cli/install#installation

Rather than recommend everyone use the (not very secure) `curl -sSL https://random-stuff | sh` method? Their docs e.g. recommend using OS package managers first, and the curl install as a lower down option.

Reply:
There are a lot of different options on that page, OS, node, etc.  I agree, it may be better to simply present the options available through a link.  We could keep the current line as an example.  My thought when I opened this was to stick as close to the original outline as possible, which uses curl.

I'm fine proceeding either way, what do you want to see @pete-woods @rosieyohannan ?

Reply:
https://circleci.atlassian.net/wiki/spaces/SD/pages/6340575858/Avoiding+curl+bash+in+the+product

Reply:
Would far rather see us encourage secure practises for our customers :)

Reply:
Updated, please re-review.

Reply:
Looks great!